docs/docsite/rst/community/maintainers_guidelines.rst

```diff
@@ -119,14 +123,82 @@ Some other general guidelines to encourage contributors:
 * If somebody suggests a good idea, mention it or put a thumbs up.
 * After merging, thank the author and reviewers for their time and effort.
 
+.. _review_checklist:
+
+Reviewing bug reports
```

Comment:
```suggestion
Reviewing bug reports
```
The reason why it can be worth having the checklist in a separate file is that all contributors can review issues/PRs, not only maintainers. Maybe we could put links to it in different places like in the contributor path doc, etc.
But i'm not sure, maybe it's OK to have it here.
Would be also nice to hear @felixfontein and @briantist 's thoughts.